### ISSUE TITLE
How to build/run parrot?

### ISSUE THREAD

**Issue by NovaLeft** [sotelo/parrot#7]
Cool paper!

But in as far as this project/code goes (in github) I see lots of dependancies, such as 'blocks', and no guide how to run/repro the claimed results.

Any chance the authors can provide a bit more info?

-best

**Comment by manugarri** [comment]
@NovaLeft I dont think this repo has any real functionality as of now. We will need to wait until the authors update it.
